packages/chat/src/styles.css

```diff
@@ -115,3 +115,25 @@
 /* .vfrc-chat-input--button {} */
 
 /* .vfrc-footer--watermark {} */
+
+.markdown h1,
```

Comment:
`globalStyle` doesn't work so added the needed styles here instead. 

Is this a really bad/dumb approach?

Reply:
should be okay since we're using the shadow DOM